packages/manager/src/layout/UserStatus.tsx

```diff
@@ -14,7 +14,6 @@ import {
 import { useHistory } from "react-router-dom";
 import UserAvatar from "../static/img/avatar.svg";
 import { useKeycloak } from "@react-keycloak/web";
```

Comment:
```suggestion
import { useKeycloak } from "@react-keycloak/web";
import { ISPAshipJWT } from "../keycloak";
```

Reply:
I just mention this on the top one, I'll move the interface to a single model, make the mock keycloak will work